blocks/social/social.js

```diff
@@ -52,42 +53,87 @@ function writeDate(timestamp) {
   return `${day} ${month}`;
 }
 
-function buildTwitterTile(tile, data) {
-  const { user } = data;
-  const profileURL = `${TWITTER_URL}${user.screen_name}`;
+function buildSocialTile(tile, data, network) {
+  const profileURL = data.user.profileUrl;
```

Comment:
my first instinct is that this is where that /undefined is coming from. based on the different formats that this feed provides instagram vs. twitter data, i'm guessing there's probably _not_ one unified path to the profile url in both twitter and facebook data, but i'd love to be wrong.

Reply:
good catch, will work on it a bit.

Reply:
was a simple fix, just has the case of the parameter wrong when I passed it into the function: `profileURL` vs. `profileUrl`